docs/source/getting-started/assessing-utility/composition.ipynb

```diff
@@ -0,0 +1,742 @@
+       "│ Actual Mean Hours Worked ┆ Actual Median Hours Worked │\n",
+       "│ ---                      ┆ ---                        │\n",
+       "│ f64                      ┆ f64                        │\n",
+       "╞══════════════════════════╪════════════════════════════╡\n",
+       "│ 37.634091                ┆ 37.0                       │\n",
+       "└──────────────────────────┴────────────────────────────┘"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "candidates = list(range(25,56))\n",
+    "\n",
+    "actual_hours = df.select(\n",
+    "    pl.col(\"HWUSUAL\").fill_null(40.0).mean().alias(\"Actual Mean Hours Worked\"), \n",
+    "    pl.col(\"HWUSUAL\").fill_null(40.0).median().alias(\"Actual Median Hours Worked\"),\n",
+    ").collect()\n",
+    "\n",
+    "actual_hours"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div><style>\n",
+       ".dataframe > thead > tr,\n",
+       ".dataframe > tbody > tr {\n",
+       "  text-align: right;\n",
+       "  white-space: pre-wrap;\n",
+       "}\n",
+       "</style>\n",
+       "<small>shape: (1, 2)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>DP Mean Hours Worked</th><th>DP Median Hours Worked</th></tr><tr><td>f64</td><td>i64</td></tr></thead><tbody><tr><td>41.350777</td><td>54</td></tr></tbody></table></div>"
+      ],
+      "text/plain": [
+       "shape: (1, 2)\n",
+       "┌──────────────────────┬────────────────────────┐\n",
+       "│ DP Mean Hours Worked ┆ DP Median Hours Worked │\n",
+       "│ ---                  ┆ ---                    │\n",
+       "│ f64                  ┆ i64                    │\n",
+       "╞══════════════════════╪════════════════════════╡\n",
+       "│ 41.350777            ┆ 54                     │\n",
+       "└──────────────────────┴────────────────────────┘"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "dp_hours = context.query().select(\n",
+    "    pl.col(\"HWUSUAL\").fill_null(40.0).dp.mean((1,45)).alias(\"DP Mean Hours Worked\"), \n",
+    "    pl.col(\"HWUSUAL\").fill_null(40.0).dp.median(candidates).alias(\"DP Median Hours Worked\"),\n",
+    ").release().collect()\n",
+    "\n",
+    "dp_hours"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Interactive Sequential Composition"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In cases where you want to apply interactive sequential composition, you can use the `dp.c.make_sequential_composition` method. There are two ways to implement sequential composition: using transformations and measurements directly and passing in polars queries. \n",
```

Comment:
"(1) using transformations and measurements directly, and (2) passing in polars queries."